bodhi/server/services/releases.py

```diff
@@ -260,6 +260,10 @@ def query_releases_json(request):
         query = query.join(Release.builds).join(Build.package)
         query = query.filter(or_(*[Package.id == p.id for p in packages]))
 
+    exclude_archived = data.get('exclude_archived')
+    if exclude_archived is not None and exclude_archived:
```

Comment:
You could simplify this:

```suggestion
    if exclude_archived:
```